.github/pull_request_template.md

```diff
@@ -1,14 +1,111 @@
----
+## Description
 
-<!-- Everything below this line will be removed from the commit message when the PR is merged -->
+<!--
 
-## PR Checklist
+The description should describe the change you're making.
+It will be used as the commit message for the squashed commit once the PR is merged.
 
--   [ ] Verify if the change requires a changeset. See [CONTRIBUTING.md](https://github.com/vivid-planet/comet/blob/HEAD/CONTRIBUTING.md)
--   [ ] Link to the respective task if one exists: <!-- For instance, COM-123 -->
--   [ ] Provide screenshots/screencasts if the change contains visual changes
+PLEASE DESCRIBE WHY YOU'RE MAKING THE CHANGE, NOT WHAT YOU'RE CHANGING.
+Reviewers see what you're changing when reviewing the code.
+However, they might not understand your motives as to why you're making the change.
 
-<details>
-    <summary>Screenshots/screencasts</summary>
-    <!-- Insert screenshots/screencasts here -->
-</details>
+Your description should include:
+-   The problem you're facing
+-   Your solution to the problem
+-   An example usage of your change
+
+--->
+
+<!--
+
+Everything below this is intended to help ease reviewing this PR.
+Remove all unrelated sections.
+
+WHEN MERGING THE PR, REMOVE THIS FROM THE COMMIT MESSAGE.
```

Comment:
We could also include everything in the squashed commit message. However, in my experience this section is often incomplete or out of date. What do you think?

Reply:
I think it is perfect as you made it